src/main/kotlin/de/cyface/collector/handler/StatusHandler.kt

```diff
@@ -56,46 +56,53 @@ class StatusHandler(private val storageService: DataStorageService) : Handler<Ro
             return
         }
         try {
-            // Check if measurement already exists in database
+            // Decide which upload strategy to use
+            uploadStrategy = if (request.path().contains("attachments")) {
```

Comment:
Same as above. Please use different handlers on separate endpoints.

Reply:
siehe Kommentar in Jira Task